cypress/integration/podcasts.spec.js

```diff
@@ -102,9 +110,21 @@ describe("newsdesk logged in", () => {
 
   it.skip("should move item between slots when dragging")
   it.skip("should move item within a slot when dragging")
-  it.skip(
-    "should add item to slot and remove from inbox when dragging from inbox",
-  )
+
+  it("should add item to slot and remove from inbox when dragging from inbox", () => {
+    cy.get("section[slotno=7] textarea").type("Remove item in inbox{enter}")
+    cy.get("section[slotno=7]").should("contain", "Remove item in inbox")
+    cy.get("section[slotno=7] button[title='Share to podcast']").click()
+    cy.get("section[slotno=7] input[id='dev2'][type='checkbox']").click()
+    switchPodCast("dev 2 sandbox")
+    cy.get("#inbox-column").should("contain", "Remove item in inbox")
+    switchPodCast("dev sandbox")
+    cy.get("section[slotno=7] ul div")
+      .eq(0)
+      .trigger("dragend", { clientX: 820, clientY: 402 }, { force: true })
+    cy.get("section[slotno=6]").should("contain", "Remove item in inbox")
+    cy.get("section[slotno=6] button[title='Delete']").click()
```

Comment:
no, this is dragging from inbox to drafts, so the code looks something more like this:
```suggestion
    cy.get("section[slotno=7] textarea").type("Remove item in inbox{enter}")
    cy.get("section[slotno=7] button[title='Share to podcast']").click()
    cy.get("section[slotno=7] input[id='dev2'][type='checkbox']").click()
    switchPodCast("dev 2 sandbox")
    cy.get("#inbox-column section[slotno=7] ul div")
      .eq(0)
      .trigger("dragend", { clientX: 820, clientY: 402 }, { force: true })
    cy.get("#inbox-column").should("not.contain", "Remove item in inbox")
```

Reply:
- Inbox dragged functionality is not implemented on this branch so your suggestion is not working for this test
- I have implemented that when an item is dragged from one slot to another slot then this item is removed from the inbox.